projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts

```diff
@@ -1,98 +0,0 @@
-import { clickAllowAllFromBanner } from '../../../helpers/anonymous-consents';
```

Comment:
Why change this file? This doesn't seem to fit the scope of the PR.
Let's discuss.